components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/AuthenticationValve.java

```diff
@@ -282,10 +282,18 @@ private void unsetCurrentSessionIdThreadLocal() {
     private boolean validateTenantDomain(Request request, Response response, String tenantDomain)
             throws IOException, ServletException {
 
+        // todo: need to decide whether to support org mgt requests with a structural org domain.
```

Comment:
Let's create a issue and remove the todo.

Reply:
https://github.com/wso2/product-is/issues/13954